src/main/java/teammates/ui/template/FeedbackResponseComment.java

```diff
@@ -40,10 +41,53 @@ public FeedbackResponseComment(FeedbackResponseCommentAttributes frc, String giv
         this.commentId = frc.getId();
         this.giverDisplay = giverDisplay;
         this.createdAt = frc.createdAt.toString();
-        this.editedAt = frc.getEditedAtText(giverDisplay.equals("Anonymous"));
+        this.editedAt = frc.getEditedAtText(isAnonymous(giverDisplay));
         this.commentText = frc.commentText.getValue();
     }
 
+    // Used in InstructorFeedbackResponseComment which is part of 
+    // InstructorFeedbackResponseCommentsLoadPageData
+    public FeedbackResponseComment(FeedbackResponseCommentAttributes frc, String giverDisplay,
+                                   String giverName, String recipientName, String extraClass,
+                                   boolean withVisibilityIcon, boolean withNotificationIcon, String whoCanSeeComment,
+                                   String showCommentToString, String showGiverNameToString,
+                                   boolean isAllowedToEditAndDeleteComment, boolean editDeleteEnabledOnlyOnHover,
+                                   boolean responseVisibleToRecipient, boolean responseVisibleToGiverTeam,
+                                   boolean responseVisibleToRecipientTeam, boolean responseVisibleToStudents,
+                                   boolean responseVisibleToInstructors) {
```

Comment:
I'm not really comfortable with the 17-variable constructor there. See if you can achieve the same objective in a different manner? One way to do is what I did in `Comment`.